webtester-core/src/main/java/info/novatec/testit/webtester/pageobjects/MultiSelect.java

```diff
@@ -190,6 +192,7 @@ public void deselectByIndices(final Integer... indices) {
             String index = option.getAttribute("index");
             allSelectedIndices.add(Integer.valueOf(index));
         }
+        this.markAsRead();
```

Comment:
all of these `this.markAsRead()` could be replaced by adding the marking to the ` public List<WebElement> getAllSelectedOptions()` method